atomic_reactor/util.py

```diff
@@ -852,4 +853,226 @@ def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, par
     return dfparser
 
 
+class Labels(object):
+    def __init__(self, df_labels):
+        """
+        Create a new Labels object
+        providing access to actual newest labels as well as old ones
+        """
+        self.LABEL_TYPE_NAME = 1
```

Comment:
The usage of `int`s imply a sequence, which is not the case here (which is correct). Consider using something like this instead to avoid confusion in the future:

```python
LABEL_TYPE_NAME = object()
LABEL_TYPE_VERSION = object()
...
```

Each call to `object()` returns a unique hashable object.